extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -391,15 +390,13 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     if (this.element.getAttribute('data-loading-strategy')) {
       return false;
     }
-    let vpRange =
-        parseInt(this.postAdResponseExperimentFeatures['render-idle-vp'], 10);
-    if (isNaN(vpRange)) {
-      if (isExperimentOn(this.win, 'dfp_ff_render_idle_launch')) {
-        vpRange = 12;
-      } else {
-        return false;
-      }
+    const expVal = this.postAdResponseExperimentFeatures['render-idle-vp'];
+    let vpRange = parseInt(expVal, 10);
+    if (expVal && isNaN(vpRange)) {
```

Comment:
can expVal ever be null or undefined? because if so,  you won't return false when I assume you would want to

Reply:
If the feature is never specified then yes it will be undefined but that's fine, it should take launched value of 12.  I needed a way that was consistent with current prod code to allow holdback behavior.